lib/micro_deploy/cli/project_generate_group.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+require 'thor'
+
+module MicroDeploy
+  module CLI
+    class ProjectGenerateGroup < Thor::Group
+      REPOSITORY_DIRECTORY = 'repository.git'
+
+      include Thor::Actions
+
+      argument :name, type: :string, desc: 'The name of the project'
+
+      def setup_git
+        system! "git init --bare #{REPOSITORY_DIRECTORY}", "Initialize empty git repository at #{REPOSITORY_DIRECTORY}"
```

Comment:
Should we add a check that git is installed before calling it?

Reply:
Yes why not?